elasticsearch7/search-index/src/main/java/org/vertexium/elasticsearch7/Elasticsearch7SearchIndex.java

```diff
@@ -404,33 +440,25 @@ void addElement(
 
         bulkUpdateService.flushUntilElementIdIsComplete(elementMutation.getId());
```

Comment:
We should wait in the BulkUpdateService when the batch is converted to a request.